omod/src/main/java/org/openmrs/module/coreapps/page/controller/conditionlist/AddConditionPageController.java

```diff
@@ -16,10 +16,28 @@
 
 import org.openmrs.ui.framework.page.PageModel;
 import org.springframework.web.bind.annotation.RequestParam;
+import org.openmrs.module.coreapps.CoreAppsConstants;
+import org.openmrs.api.context.Context;
 
 public class AddConditionPageController {
 	
 	public void controller(PageModel model, @RequestParam(value = "returnUrl", required = false) String returnUrl) {
 		model.addAttribute("returnUrl", returnUrl);
+		String criterion = Context.getAdministrationService().getGlobalProperty(CoreAppsConstants.CONDITIONS_TO_BE_DISPLAYED);
+	    String criterion_array[] = criterion.split(",");
+		String conditionListClasses= "";
+		for(String i : criterion_array) 
+		{
+			if(i.equalsIgnoreCase("Diagnosis"))
+			conditionListClasses += "8d4918b0-c2cc-11de-8d13-0010c6dffd0f,";
+			else if(i.equalsIgnoreCase("Symptom"))
+			conditionListClasses += "8d492954-c2cc-11de-8d13-0010c6dffd0f,";
+			else if(i.equalsIgnoreCase("Symptom/Finding"))
+			conditionListClasses += "8d492b2a-c2cc-11de-8d13-0010c6dffd0f,";
+			else
+			conditionListClasses += "8d491a9a-c2cc-11de-8d13-0010c6dffd0f,";
+		}  
+		conditionListClasses = conditionListClasses.substring(0,conditionListClasses.length()-1);
```

Comment:
Can we put a space after the comma parameter?

Reply:
i will update it